scripts/initializers/company-switcher.js

```diff
@@ -0,0 +1,51 @@
+import { getHeaders } from '@dropins/tools/lib/aem/configs.js';
+import { initializers } from '@dropins/tools/initializer.js';
+import { initialize, setFetchGraphQlHeaders } from '@dropins/storefront-company-switcher/api.js';
+import * as pdpFetchGraphQl from '@dropins/storefront-pdp/api.js';
+import * as cartFetchGraphQl from '@dropins/storefront-cart/api.js';
+import * as searchFetchGraphQl from '@dropins/storefront-product-discovery/api.js';
+import * as orderFetchGraphQl from '@dropins/storefront-order/api.js';
+import * as accountFetchGraphQl from '@dropins/storefront-account/api.js';
+import * as companyFetchGraphQl from '@dropins/storefront-company-switcher/api.js';
+import * as checkoutFetchGraphQl from '@dropins/storefront-checkout/api.js';
+import { initializeDropin } from './index.js';
+import { fetchPlaceholders } from '../commerce.js';
+
+const companyHeader = 'X-Adobe-Company';
+const customerGroupHeader = 'Magento-Customer-Group';
+const companySessionStorageKey = 'DROPIN__COMPANYSWITCHER__COMPANY__CONTEXT';
+const groupSessionStorageKey = 'DROPIN__COMPANYSWITCHER__CUSTOMER_GROUP__CONTEXT';
```

Comment:
I understand the desire to allow these to be configurable - but can we just default these internally in the dropin, if they are not passed to the initializer config? We can document them as an option, but since these are already default values, it doesn't really need to be exposed here.

Generally I try to reduce the amount of code/information we are putting in boilerplate.